streams/src/test/java/org/apache/kafka/streams/integration/KTableKTableForeignKeyJoinIntegrationTest.java

```diff
@@ -59,6 +60,7 @@
 
 
 @RunWith(Parameterized.class)
+@Tag("integration")
```

Comment:
This module is still using Junit 4, but `@Tag` seems to be a Junit 5 class? this seems not to take effect, is there a similar keyword in Junit 4?

Reply:
@dengziming is there something stopping us from entirely moving to JUnit 5 other than a potentially big refactor?

Reply:
No, we already have a JIRA for this: https://issues.apache.org/jira/browse/KAFKA-7342, you could take it if you are interested.

Reply:
Brilliant, thank you. I will probably give it a go. I have picked up a couple of other JIRA tickets so I would like to finish them first.

Reply:
Thanks for looking into this @dengziming 

Ah! My bad. `@Category({IntegrationTest.class})` should work with Junit 4. See: https://github.com/apache/kafka/blob/trunk/build.gradle#L470

I found these test running with the command `./gradlew unitTest` based on visual inspection. I have also tried to explore the idea of how to automatically detect if an integration test is running as a unit test but I don't have any solution. The primary blocker is that many files with suffix as "IntegrationTest" actually run as UnitTests (and I think that is expected?)